DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Services;
+using Google.Apis.Sheets.v4;
+using Google.Apis.Sheets.v4.Data;
+using static Google.Apis.Sheets.v4.SpreadsheetsResource.ValuesResource;
+
+namespace DiscordBot.DataAccess
+{
+    public class EventsSheetsService
```

Comment:
I would expect this to have an interface as well, so that it can be used by the rest of the project 🙂

Reply:
Yep I believe Frank (@kwunkiuma) is in the process of writing one, we were going to put that up for review once this had gone through if that is ok? We could add it now otherwise 🙂

Reply:
Ah okay. For next time I would recommend creating the interface first, or creating it with the implementation of the class, so that it's clear how it will be used. I think one of these approaches would work well:
* Person A and person B discuss which methods are needed in the interface
* Person A implements the service class including the interface
* In the meantime person B uses hardcoded data instead of the service class
* Whoever merges last deals with replacing the hardcoded data with the real values from the service class

or:
* Person A and person B discuss which methods they want on the interface
* Person A or person B open a PR with the interface + a dummy implementation (returning hardcoded values)
* Person A implements the service class
* Person B uses the dummy implementation of the service class in the meantime

Does that make sense? 🙂

Reply:
Yeah that makes sense. I was expecting to have more trouble implementing a basic Google Sheets API than I did so the original plan was for Frank to have had that interface made before I had anything worth pushing. 

That said, maybe I should have done some more work on this before I put it up for review, waiting for the interfaces (/helping Frank write them) and implementing them all instead of just pushing the skeleton of a service with no actual features. I could help Frank with writing the interfaces and implement this as part of this PR if you like? 🙂

Reply:
I think putting something up for review now was the right call, but for future reference I think the interface for this class should have been agreed first as part of planning. It should just be one interface for this service as far as I understand, no need to agree any more interfaces. 

For right now I think the best thing to do is to get this merged then prioritise getting an interface up for review with mock implementations. Then you can focus on implementing that interface properly and @kwunkiuma can focus on using it. Does that make sense? 🙂

Reply:
There was a bit of miscommunication with this, we had agreed on a general outline of methods that the service should provide but we both assumed that the other person would include it in their PR. We've sorted out the misunderstanding now and Benji has a few methods to work on implementing while I focus more on the conversation side with the bot.

Reply:
Sounds good! 🙂